ksql-engine/src/test/java/io/confluent/ksql/parser/KsqlParserTestUtil.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License; you may not use this file
+ * except in compliance with the License.  You may obtain a copy of the License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.parser;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasSize;
+
+import io.confluent.ksql.metastore.MetaStore;
+import io.confluent.ksql.parser.KsqlParser.PreparedStatement;
+import io.confluent.ksql.parser.tree.Statement;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public final class KsqlParserTestUtil {
```

Comment:
This class is duplicated in the `ksql-parser` module. Is there a way to avoid the duplication and have a single test util class?

Reply:
Ok. The class is not that large, so it would be fine to have it duplicated for now.